cassandra/src/main/scala/org/apache/zeppelin/cassandra/CassandraInterpreterHelper.scala

```diff
@@ -0,0 +1,395 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.cassandra
+
+import java.net.InetAddress
+import java.nio.ByteBuffer
+import java.util
+import java.util.Date
+
+import com.datastax.driver.core.DataType.Name._
+import com.datastax.driver.core._
+import com.datastax.driver.core.exceptions.DriverException
+import com.datastax.driver.core.policies.{LoggingRetryPolicy, FallthroughRetryPolicy, DowngradingConsistencyRetryPolicy, Policies}
+import org.apache.zeppelin.cassandra.TextBlochHierarchy._
+import org.apache.zeppelin.display.Input.ParamOption
+import org.apache.zeppelin.interpreter.InterpreterResult.Code
+import org.apache.zeppelin.interpreter.{InterpreterException, InterpreterResult, InterpreterContext}
+import org.slf4j.LoggerFactory
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
+
+
+case class CassandraQueryOptions(consistency: Option[ConsistencyLevel],
+                                 serialConsistency:Option[ConsistencyLevel],
+                                 timestamp: Option[Long],
+                                 retryPolicy: Option[RetryPolicy],
+                                 fetchSize: Option[Int])
+
+object CassandraInterpreterHelper {
+  
+  val CHOICES_SEPARATOR : String = """\|"""
+  val VARIABLE_PATTERN = """\{\{[^}]+\}\}""".r
+  val SIMPLE_VARIABLE_DEFINITION_PATTERN = """\{\{([^=]+)=([^=]+)\}\}""".r
+  val MULTIPLE_CHOICES_VARIABLE_DEFINITION_PATTERN = """\{\{([^=]+)=((?:[^=]+\|)+[^|]+)\}\}""".r
+
+  val defaultRetryPolicy = Policies.defaultRetryPolicy()
+  val downgradingConsistencyRetryPolicy = DowngradingConsistencyRetryPolicy.INSTANCE
+  val fallThroughRetryPolicy = FallthroughRetryPolicy.INSTANCE
+  val loggingDefaultRetryPolicy = new LoggingRetryPolicy(defaultRetryPolicy)
+  val loggingDownGradingRetryPolicy = new LoggingRetryPolicy(downgradingConsistencyRetryPolicy)
+  val loggingFallThrougRetryPolicy = new LoggingRetryPolicy(fallThroughRetryPolicy)
+
+  val preparedStatements : mutable.Map[String,PreparedStatement] = mutable.Map[String,PreparedStatement]()
+
+  val logger = LoggerFactory.getLogger(classOf[CassandraInterpreterHelper])
+
+  val paragraphParser = new ParagraphParser
+  val boundValuesParser = new BoundValuesParser
+
+  def toScalaList[A](list: java.util.List[A]): List[A] = list.asScala.toList
+  def toJavaList[A](list: List[A]): java.util.List[A] = list.asJava
+}
+
+class CassandraInterpreterHelper(val session: Session)  {
+ 
+  import CassandraInterpreterHelper._
+
+  def interpret(session:Session, stringStatements : String, context: InterpreterContext): InterpreterResult = {
+
+    logger.info(s"Executing CQL statements : \n\n$stringStatements\n")
```

Comment:
a secondary named logged only for statements would be nice and allow to log this and the cassandra options at the debug level